### PR TITLE
Add entry points for both Admin and Widget elm apps

### PR DESCRIPTION
**Pull request by breezyboa**
Addresses #72

### CODE REVIEW

**Comment by breezyboa** [comment]
@sudeeptarlekar @vipulnsward  I've added an entry point for Admin
Webpack now compiles both widget and admin as separate js files
I think there may be some rails changes to be added to get /admin path working
@piyushtiw can you help with this?

**Comment by vipulnsward** [comment]
@sudeeptarlekar can take care of them

**Comment by vipulnsward** [comment]
@breezyboa instead of "Widget" lets move to "Client" namespace.

Admin vs Client // Client can them comprise of more than just a "Widget"

**Comment by breezyboa** [comment]
renamed widget to client
@sudeeptarlekar _a assigning it to you

**Comment by sudeeptarlekar** [comment]
Looks Good. Merging

File: app/assets/stylesheets/client/reset.scss [hunk 1]
```diff
@@ -2,7 +2,8 @@
 #acehelp-hook {
     
     text-align: left;
-
+    font-family: calibri, helvetica, arial, sans-serif;
```
Comment (vipulnsward) [inline app/assets/stylesheets/client/reset.scss]: Why was this added?
